### ISSUE TITLE
vue/max-attribute-per-line Error

### ISSUE THREAD

**Issue by rashadrivera** [vuejs/eslint-plugin-vue#1628]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 6.7.2
- **vue version:** 2.6.11
- **eslint-plugin-vue version:** 6.2.2
- **Node version:** 14.17.6
- **Operating System:** Windows 10 20H2 (19042.1237)

**Please show your full configuration:**

```json5
module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: [
    'plugin:vue/essential',
    //'@vue/airbnb',
    '@vue/typescript/recommended',
  ],
  parserOptions: {
    ecmaVersion: 2020,
  },
  rules: {
    'vue/max-attributes-per-line': ['error', { singleline: { max: 1, allowFirstLine: true }, multiline: { max: 1, allowFirstLine: true }}],

    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
  },
};
```

**What did you do?**	

```vue
<template>
  <div id="app">
    <img alt="Vue logo"
         src="./assets/logo.png">
    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App"/>
  </div>
</template>
```

**What did you expect to happen?**
I expect that I won't get a compilation error.

**What actually happened?**
 ERROR  Failed to compile with 1 error                                                                                                                                     9:31:30 PM

```
Syntax Error: Error: .eslintrc.js:
        Configuration for rule "vue/max-attributes-per-line" is invalid:
        Value {"max":1,"allowFirstLine":true} should be number.
        Value {"max":1,"allowFirstLine":true} should NOT have additional properties.
        Value {"max":1,"allowFirstLine":true} should match some schema in anyOf.
    at Array.forEach (<anonymous>)
```
**Repository to reproduce this issue**
https://github.com/rashadrivera/lint-issue_max-attributes-per-line

**Comment by ota-meshi** [comment]
You are using the old eslint-plugin-vue.  Please use the latest version.

**Comment by ferrykranenburgcw** [comment]
I am using version `8.0.3` and still have this issue. What am I doing wrong?

**Comment by ferrykranenburgcw** [comment]
This is my configuration:

```
{
  "root": true,
  "env": {
    "browser": true,
    "es6": true,
    "node": true
  },
  "parserOptions": {
    "ecmaVersion": 2021
  },
  "rules": {
    // Override our default settings just for this directory
    "eqeqeq": 0,
    "strict": 0,
    "linebreak-style": 0,
    // Attributes on new lines
    "vue/max-attributes-per-line": [
      "error",
      {
        "singleline": {
          "max": 3,
          "allowFirstLine": true
        },
        "multiline": {
          "max": 3,
          "allowFirstLine": true
        }
      }
    ],
    // let us use @ts-ignore
    "@typescript-eslint/ban-ts-ignore": 0,
    // This is problematic in refactoring withing Intellij
    "vue/attribute-hyphenation": 0,
    // We use v-html
    "vue/no-v-html": 0,
    // Disable self-closing tags
    "vue/html-self-closing": 0,
    // We do not care about the order strictly
    "vue/component-tags-order": 0
  },
  "extends": [
    "plugin:vue/recommended",
    "prettier/prettier"
  ]
}
```

**Comment by ferrykranenburgcw** [comment]
Sorry, it is a breaking change: https://github.com/vuejs/eslint-plugin-vue/releases/tag/v8.0.0. Documentation should be updated: https://eslint.vuejs.org/rules/max-attributes-per-line.html#vue-max-attributes-per-line

**Comment by ota-meshi** [comment]
Thank you for letting me know. I will update the docs.

**Comment by code1x1** [comment]
https://github.com/nextcloud/eslint-config/issues/253

**Comment by code1x1** [comment]
I am using version `8.2.0`.  running into the same error message

**Comment by FloEdelmann** [comment]
@code1x1 see my comment there: https://github.com/nextcloud/eslint-config/issues/253#issuecomment-992496241

There's nothing to be done in `eslint-plugin-vue`.

**Comment by designermonkey** [comment]
Any explanation as to why `allowFirstLine` was removed?

**Comment by rashadrivera** [comment]
> You are using the old eslint-plugin-vue. Please use the latest version.

Hello,

I'm still seeing this issue.  I just created a brand new Vue3 sample project and added this rule.  The same exact configuration is still showing.  I which I could share the project so you'd have all the answers at once. 

![image](https://user-images.githubusercontent.com/24625378/159134544-1b98481d-ae6a-4d02-a4f2-b7e5a0757364.png)

**Comment by ota-meshi** [comment]
`vue/first-attribute-linebreak` rule has been added to enforce the location of the first attribute, so that option is no longer needed.

https://eslint.vuejs.org/rules/first-attribute-linebreak.html
